packages/react/src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -18259,7 +18265,9 @@ Map {
       "editOptionText": Object {
         "type": "string",
       },
-      "hasMultiValue": [Function],
```

Comment:
I assume this is the same as `addToList`?

Reply:
yes.